client/post-editor/editor-ground-control/test/index.jsx

```diff
@@ -181,7 +181,7 @@ describe( 'EditorGroundControl', function() {
 				/>
 			).instance();
 
-			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Publish' );
+			expect( tree.getPrimaryButtonLabel() ).to.equal( 'Update' );
```

Comment:
Confused me at first, but I think I understand the original expectation. When creating a new post, if the user schedules the post in the past (while still drafted), we'd expect the text to be "Publish", not "Update". The latter is true only when the post is already published.

![schedule](https://cloud.githubusercontent.com/assets/1779930/14787937/c19d6ae8-0ad3-11e6-9339-71ebc2364a7b.gif)